src/v2i-hub/CDASimAdapter/src/CDASimAdapter.cpp

```diff
@@ -66,22 +66,26 @@ namespace CDASimAdapter{
             }else {
                 PLOG(logERROR) << "CDASim connection failed!" << std::endl;
             }
-            
+
         }
     }
 
 
 
     void CDASimAdapter::forward_time_sync_message(tmx::messages::TimeSyncMessage &msg) {
+
         std::string payload =msg.to_string();
-        PLOG(logDEBUG1) << "Sending Time Sync Message " << msg << std::endl;
+        auto time_now = std::chrono::system_clock::now();
```

Comment:
Can we put this entire logic inside a block that is skipped if not in debug mode and add a comment here indicating what this is for and maybe that it will eventually be removed.